src/main/webapp/app/markdown-editor/commands/bold.command.ts

```diff
@@ -15,36 +15,14 @@ export class BoldCommand extends Command {
     execute(): void {
         const selectedText = this.getSelectedText();
         let textToAdd = '';
-        let startSpace = '';
-        let endSpace = '';
-        let startIndx = 0;
-        let endIndx = 0;
-        const selectedTextLength = selectedText.length;
 
-        if (selectedText.includes('**')) {
+        if (selectedText.charAt(0) === '*' && selectedText.charAt(selectedText.length - 1) === '*') {
```

Comment:
Good fix!
However I think it should be "**" instead of "*" here, right?

Otherwise it is the same as in https://github.com/ls1intum/Artemis/pull/937/files#diff-a6bae12e2a7d3f19137d1bb31cab1fb9R19.